utils/logger.go

```diff
@@ -9,24 +9,30 @@ import (
 // and logJSON set
 func GetLogger(bind string, port, verbosity int, logJSON bool) logrus.FieldLogger {
 	log := logrus.New()
+
 	switch verbosity {
 	case 0:
-		log.Level = logrus.InfoLevel
+		log.Level = logrus.ErrorLevel
```

Comment:
I think that the verbosity code was added here by mistake. I added this hierarchy to make more sense to devs who want to use this option.